### PR TITLE
added depends_on in docker-compose file

### PR DESCRIPTION
**Pull request by adrean-hipe**
OVERVIEW

- added depends_on line in the docker-compose file
- added the version '3'
- changed phpmyadmin:latest to phpmyadmin/phpmyadmin

NOTES

- I was getting an error: 
   ERROR: The Compose file './docker-compose.yml' is invalid because: Unsupported config option for services: 'phpmyadmin'
   when performing 'docker-compose build' and 'docker-compose up -d'
- This happened after updating the docker software
- After the changes, the commands run properly

image

### CODE REVIEW

**Review by bpocrafael — APPROVED** [review]
lgtm

File: docker-compose.yml
Comment (hipe01mark) [inline docker-compose.yml:1]: Where's the webserver here?

Comment (adrean-hipe) [reply docker-compose.yml:1]: we used FROM php:8.1.0-apache included in the Dockerfile sir, do we also include this in our docker-compose file?

**Review by hipe-francis — APPROVED** [review]
LGTM 🚀

File: docker-compose.yml [hunk 1]
```diff
@@ -25,3 +25,24 @@ services:
...
+      - 8080:80
+    depends_on:
+      - laravel-docker
+
+    # Mod Rewrite
+    command: ["apache2-foreground"]
+    links:
+      - laravel-docker
+    networks:
+      - laravel-network
```
Comment (hipe01mark) [inline docker-compose.yml:44]: remove the comment and align the line space to web server service

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇 I have a minor comment. tag it as resolve first before merging it.

File: docker-compose.yml (hunk 1 above)
Comment (adrean-hipe) [reply docker-compose.yml]: Removed comment and aligned the line space sir. Thank you ❤️

**Pull request merged**